tests/locale/keymap_or_locale.pm

```diff
@@ -18,8 +18,7 @@ use utils;
 sub verify_default_keymap_textmode {
     my ($test_string, $tag, %tty) = @_;
 
-    # test keymap in login prompt of generally unused tty3 and typing test string wait 3 seconds
-    defined($tty{console}) ? select_console($tty{console}) : send_key('alt-f3', 3);
+    defined($tty{console}) ? select_console($tty{console}) : send_key('alt-f3');
```

Comment:
Now it could be that there is missing synchronisation between this keyboard-shortcut and the typing in the next line. Can you explain why we need this manual "send_key" instead of the select_console anyway? In which cases have you seen the tty{console} to be not defined?